sw360-single/sw360-install.sh

```diff
@@ -138,14 +138,16 @@ if [ -z $SW360_source ]; then
         if [ -d "$wd/.git/" ]; then
             cd "$wd"
             git checkout $SW360_branch || git checkout -b $SW360_branch
-            git pull $SW360_gitURL $SW360_branch
+            git fetch
+            git reset --hard origin/$SW360_branch
         else
             git clone -b $SW360_branch --single-branch $SW360_gitURL "$wd"
         fi
     else
         if [ -d "$wd/.git/" ]; then
             cd "$wd"
-            git pull $SW360_gitURL
+            git fetch
+            git reset --hard origin/master
```

Comment:
please do not reset any current branch to `origin/master` , but rather to `origin/{current_branch}`

Reply:
Added new variable `currentBranch` and set value to `git rev-parse --abbrev-ref HEAD`